content/applications/hr/payroll.rst

```diff
@@ -1,18 +1,359 @@
-:nosearch:
-:show-content:
-:hide-page-toc:
-:show-toc:
+=====================
+Payroll configuration
+=====================
```

Comment:
I'm suggesting removing "_configuration_" from the H1 since this is now the top-level doc for the app– configuration is implied IMO.
```suggestion
=======
Payroll
=======
```

Reply:
OMG its the simplest things that I don't see =D. Great catch/idea, @samueljlieber! On to final review with @StraubCreative now.

Reply:
@StraubCreative - User Docs Structure Review was automatically tagged in this PR, but I wasn't sure if they need to be or not. I didn't think so, so I removed them.

Reply:
Yea whenever we make structural changes UDS will be notified automatically, @larm-odoo 
It's a good idea anyway that we get their eyes on changes like this :)

Thanks for taking a peek @jcs-odoo 🍻